internal/pkg/mqtt/client_options.go

```diff
@@ -53,22 +42,32 @@ type ConnectionOptions struct {
 
 // MQTTClientOptions contains the client options which are loaded via reflection
 type MQTTClientOptions struct {
-	Username          string
-	Password          string
-	ClientId          string
+	// Client Identifiers
+	Username string
+	Password string
+	ClientId string
+	// Connection information
 	Topic             string
 	Qos               int
-	KeepAlive         int
+	KeepAlive         int // Seconds
 	Retained          bool
 	ConnectionPayload string
```

Comment:
What is `ConnectionPayload ` used for?

Reply:
I removed this configuration since it was not being propagated to the underlying client anyway. I cannot find any documentation regarding this property, but I believe the client sends this specified `string` to the boker upon a successful connection.